universe/assets_3d/api_assets_3d.go

```diff
@@ -0,0 +1,318 @@
+package assets_3d
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+
+	"github.com/momentum-xyz/ubercontroller/types/entry"
+	"github.com/momentum-xyz/ubercontroller/universe"
+)
+
+func (a *Assets3d) apiCreateAsset3d(c *gin.Context) {
+	in := struct {
```

Comment:
in future we will move all request struct outside of methods, so it better to give more meaningful names to you input, right now it's ok just to specify, for example `inBody` or `inQuery`